mobile/src/main/java/org/openhab/habdroid/ui/activity/AbstractWebViewFragment.kt

```diff
@@ -202,7 +206,7 @@ abstract class AbstractWebViewFragment : Fragment(), ConnectionFactory.UpdateLis
         val context = context ?: return
         askForShortcutTitle(context, shortcutInfo) {
             val success = ShortcutManagerCompat.requestPinShortcut(context, it, null)
-            GlobalScope.launch {
+            launch {
```

Comment:
Is this `launch` call actually needed (anymore)? AFAICT the callbakc should be launched in the main thread anyway.

Reply:
I remove the `launch` and used toasts below. On my device the app is closed directly after pinning a shortcut and
therefore the Snackbar is shown, but never seen by the user.

Reply:
Maybe it's not a real issue when the success snackbar isn't shown: My device minimizes the app, scrolls to the page where the shortcut is and animates it. This way the user known that the shortcut has been created.

Reply:
Reverted

Reply:
I reverted the change to Toasts. The error snackbar is also shown without the `launch` before.